### PR TITLE
Kotlin 5.0 release

### PR DESCRIPTION
**Pull request by rachel-mack**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kotlin/blob/master/REVIEWING.md)

JIRA tickets - https://jira.mongodb.org/browse/JAVA-5292?jql=project%20%3D%20%2210006%22%20AND%20fixVersion%20%3D%205.0.0%20AND%20text%20~%20%22kotlin%22
Staging - <https://docs-mongodbcom-staging.corp.mongodb.com/kotlin/docsworker-xlarge/kotlin-5.0-release/whats-new/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -12,14 +12,27 @@ What's New
...
 
-.. _upcoming-breaking-changes:
+.. _version-5.0:
 
-Upcoming Breaking Changes
--------------------------
+What's New in 5.0
+------------------
+
+New features of the 4.11 driver release include:
```
Comment (mongoKart) [inline source/whats-new.txt:24]:
```suggestion
New features of the 5.0 driver release include:
```

File: source/whats-new.txt [hunk 2]
```diff
@@ -12,14 +12,27 @@ What's New
...
+
+- The ``KotlinSerializerCodecProvider`` constructor now accepts
+  ``serializersModule`` and ``bsonConfiguration`` objects:
+
+  .. code-block:: cpp
```
Comment (mongoKart) [inline source/whats-new.txt:29]:
```suggestion
  .. code-block:: kotlin
```

File: source/includes/mongodb-compatibility-table-kotlin.rst [hunk 3]
```diff
@@ -16,7 +16,7 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
-   * - 4.11
```
Comment (mongoKart) [inline source/includes/mongodb-compatibility-table-kotlin.rst:19]: Q: Should 4.11 still be included on this page?

**Review by mongoKart — CHANGES_REQUESTED** [review]
a few small things

File: source/includes/mongodb-compatibility-table-kotlin.rst (hunk 3 above)
Comment (rachel-mack) [reply source/includes/mongodb-compatibility-table-kotlin.rst:19]: Opps.

File: source/whats-new.txt [hunk 4]
```diff
@@ -12,14 +12,27 @@ What's New
...
+  ``serializersModule`` and ``bsonConfiguration`` objects:
+
+  .. code-block:: kotlin
+     
+     KotlinSerializerCodec.create(clazz.kotlin, serializersModule=serializersModule, bsonConfiguration=bsonConfiguration)
+  
+  This makes it easier to customize your configuration.
 
-- Beginning with v5.0, the {+driver-short+} requires Java 11 or later.
+- Fixes a bug that resulted in type erasure during reflection and return.
```
Comment (rozza) [inline source/whats-new.txt:35]: Nit: you can probably remove "and return".
Suggest: Fixes a Kotlin reflection bug that resulted in container type erasure.

**Review by rozza — CHANGES_REQUESTED** [review]
One very minor nit + suggestion.

**Review by rozza — APPROVED** [review]
LGTM!

**Pull request merged**